org.dash.electrum.electrum_dash.metainfo.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It should be upstreamed.

Reply:
Not sure what I should change, reading docs.

Reply:
Do you mean appdata should be added to upstream?

Reply:
Yes, that would be perfect.

Reply:
I'll ask to include AppData in the next relase, in https://github.com/akhavr/electrum-dash/issues/92